bin/interactive-install.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -eu
+shopt -s extglob
+
+echo -e "\e[32mWelcome to the secureCodeBox!\e[0m"
+echo -e "This interactive installation script will guide you through all the relevant installation steps in order to have you ready to scan."
+
+echo -e "Start ? [y/n]"
+read -r line
+if [[ $line == *[Yy] ]]; then
+  echo -e "Starting!"
+else
+  exit
+fi
+
+echo
+echo -e "Checking kubectl.."
+kube=$(kubectl version)
+if [[ $kube == *"GitVersion"* ]]; then
+  echo -e "\e[32mIt's there!\e[0m"
+else
+  echo -e "\e[31mKubectl not found, quitting..\e[0m"
+  exit
+fi
+
+echo
+echo -e "Creating namespace securecodebox-system"
+kubectl create namespace securecodebox-system || echo -e "Namespace already exists.. "
+
+if [[ $PWD == *"bin" ]]; then
```

Comment:
Here you should use a more reliable way. It is possible that someone executes this script from somewhere else, eg.:

```bash
cd /opt/local/bin
/path/to/repo/bin/interactive-install.sh
```

A good reliable way to find out where you are:

```bash
# @see: http://wiki.bash-hackers.org/syntax/shellvars
[ -z "${SCRIPT_DIRECTORY:-}" ] \
    && SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )" \
    && export SCRIPT_DIRECTORY
```

This also works if someone softlinks your script (eg. into `$HOME/bin`). You can use `dirname($SCRIPT_DIRECTORY)` as base direcotry.